content/applications/productivity/iot/config/connect.rst

```diff
@@ -1,81 +1,89 @@
-===================================
-Connect an IoT Box to your database
-===================================
+=======================================
+Connect an IoT Box to the Odoo database
+=======================================
 
-Install the Internet of Things (IoT) App on your Odoo Database.
+Install the :menuselection:`Internet of Things (IoT)` App on the Odoo database.
 
 .. image:: connect/connect01.png
    :align: center
+   :alt: The Internet of Things (IoT) App on the Odoo database.
 
-Go in the IoT App and click on Connect on the IoT Boxes page.
+Go to the :menuselection:`IoT` App and click on :guilabel:`Connect`` on the
+:abbr:`IoT (Internet of Things)` IoT Boxes page.
 
-.. image:: connect/connect02.png
+.. image:: connect/connect09.png
    :align: center
+   :alt: Connecting an IoT box to the Odoo database.
 
-Follow the steps to connect your IoT Box.
+Follow the steps to connect the IoT Box.
```

Comment:
Based on previous experiences, I would mention that people have to follow either the Ethernet version or the Wifi version. People that do not know what is the use of an Ethernet cable will try both options :'D

Reply:
Noted, great idea. I've made this change. Thanks.